deploy/ods-pipeline/charts/tasks/templates/task-ods-package-image.yaml

```diff
@@ -51,6 +51,10 @@ spec:
       description: Reference of the image stream buildah will produce. If not set, the value of `.ods/component` is used.
       type: string
       default: ''
+    - name: extra-tags
+      description: Additional image tags used (e.g. 'latest dev') when building image. The primary tag is based on the commit sha.
```

Comment:
Why did you decide to parse this with shlex? Does that allow other formats to specify the tags, like comma separated?

Reply:
This is copied from code here https://github.com/opendevstack/ods-pipeline/blob/4d5424bcdde4f008fa9e25a1dd4b61311e411461/cmd/build-push-image/buildah.go#L32 to make it behave in the same way although I haven't concluded that this is the right thing to do here. This PR did not work as is that is providing labels separated by spaces before https://github.com/opendevstack/ods-pipeline/blob/23e63cb912b760ef69cfaf48cb0f9d41a2a9d0ab/deploy/ods-pipeline/charts/tasks/templates/task-ods-package-image.yaml#L152 was added. Without this only the first label was picked up. I wonder whether the other extra-parameters using shlex are affected as well.

Reply:
Using single quotes in the parameter value definition (in the test) made this work without the quotes.